app/views/unattended/provisioning_templates/iPXE/ipxe_intermediate_script.erb

```diff
@@ -3,6 +3,8 @@ kind: iPXE
 model: ProvisioningTemplate
 name: iPXE intermediate script
 snippet: false
+description: |
+  TODO
```

Comment:
@lzap could you please provide a suggestion?

Reply:
This template is returned when `?bootstrap=1` URL parameter is provided.
It is used in pure iPXE environment and it's purpose is to perform another iPXE template call,
but with MAC address(es) of the server in order to find appropriate host record in the inventory.
For more info visit:
https://docs.theforeman.org/nightly/Provisioning_Guide/index-foreman-el.html#Configuring_Networking-Configuring_gPXE_to_Reduce_Provisioning_Times